application/models/artifact_builder.py

```diff
@@ -49,20 +40,50 @@ def _add_tags_to_neo(self, tags):
             self.neo.tags.connect(created_tag)
         self.neo.save()
 
-    def _overwrite_tags_in_neo(self, tags):
-        self.neo.tags.disconnect_all()
-        self._add_tags_to_neo(tags)
+    def _add_user_tags_to_neo(self, user_tags):
+        for tag in user_tags:
+            print(self.neo)
+            print(self.neo.user_tags)
```

Comment:
more prints to remove! :D

Reply:
later, once testing etc is done. currently needed so I can see what is happening^^